learning_resources/utils.py

```diff
@@ -89,30 +89,27 @@ def get_year_and_semester(course_run):
     return year, semester
 
 
-def semester_year_to_date(semester, year, ending=False):  # noqa: FBT002
+def semester_year_to_date(semester, year):
     """
     Convert semester and year to a rough date
 
     Args:
         semester (str): Semester ("Fall", "Spring", etc)
         year (int): Year
-        ending (boolean): True for end of semester, False for beginning
 
     Returns:
         datetime: The rough date of the course
     """
-    if semester is None or year is None:
+    if year is None:
         return None
-    if semester.lower() == "fall":
-        month_day = "12-31" if ending else "09-01"
+    if semester is None or semester in ("spring", "january_iap"):
```

Comment:
dont have enough context around the "january_iap" key but is it expected to have it as snake_cased here as opposed to [here](https://github.com/mitodl/mit-open/pull/514/files#diff-e84b4fc23d9aeac185e204b5f68f024d204b89546e680116c0f8c420d3eccb39R61)? My guess is that the referenced testcase with "January iap" used to excercise this conditional whereas now the test passes only because the default case of ```month_day = "01-01"``` is the same as the "january_iap" case